styles/pythonanywhere/catppuccin.user.css

```diff
@@ -0,0 +1,1330 @@
+/* ==UserStyle==
+@name PythonAnywhere Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/pythonanywhere
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/pythonanywhere
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/pythonanywhere/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Apythonanywhere
+@description Soothing pastel theme for PythonAnywhere
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@import url("https://python.catppuccin.com/pygments/catppuccin-variables.important.css");
```

Comment:
This doesn't go here.

Reply:
does this go to both the main website/blog?

Reply:
Yes, as per the directions on https://github.com/catppuccin/userstyles/blob/main/docs/tips-and-tricks.md#pygments.